### ISSUE TITLE
DragonFly and "Failure... No filesystem access enabled"

### ISSUE THREAD

**Issue by noloader** [randombit/botan#887]
Working on a BSD derivative called [DragonFly](https://www.dragonflybsd.org/download/).

```
$ LD_LIBRARY_PATH=. ./botan-test
Testing Botan 2.0.1 (unreleased, revision git:863b7ba99f2014b76e0ba2e2b256d0870301199b, distribution unspecified)
Starting tests rng:HMAC_DRBG with seed '14A4ABFF46570F14'
block ran 1 tests 1 FAILED
Failure 1: No filesystem access enabled.
stream ran 1 tests 1 FAILED
Failure 1: No filesystem access enabled.
hash ran 1 tests 1 FAILED
Failure 1: No filesystem access enabled.
mac ran 1 tests 1 FAILED
Failure 1: No filesystem access enabled.
modes ran 1 tests 1 FAILED
Failure 1: No filesystem access enabled.
pbkdf ran 1 tests 1 FAILED
Failure 1: No filesystem access enabled.
...
```

-----

```
$ uname -a
DragonFly dragonfly 4.6-RELEASE DragonFly v4.6.1-RELEASE #3: Sun Oct 16 21:13:25 EDT 2016     root@www.shiningsilence.com:/usr/obj/home/justin/release/4_6/sys/X86_64_GENERIC  x86_64
$ uname -m
x86_64
$ uname -s
DragonFly
```

**Comment by randombit** [comment]
Should be fixed by f8a1e6b91

**Comment by noloader** [comment]
Confirmed, cleared.
